### PR TITLE
Update benchmark-tidb-using-tpcc.md (#3354)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #3354 to release-3.1

---

### What is changed, added or deleted? (Required)
Update benchmark-tidb-using-tpcc.md

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: benchmark/benchmark-tidb-using-tpcc.md [hunk 1]
```diff
@@ -70,23 +70,49 @@ ant
...
 | DISK | Optane 500GB SSD |
 
-因为该型号 CPU 是 NUMA 架构，建议先用 `taskset` 进行绑核，首先用 `lscpu` 查看 NUMA node，比如：
+因为该型号 CPU 是 NUMA 架构，建议用 `numactl` 进行绑核。
 
-```text
-NUMA node0 CPU(s):     0,2,4,6,8,10,12,14,16,18,20,22,24,26,28,30,32,34,36,38
-NUMA node1 CPU(s):     1,3,5,7,9,11,13,15,17,19,21,23,25,27,29,31,33,35,37,39
-```
+1. [安装 numactl 工具](/check-before-deployment.md#安装-numactl-工具) 。
```
Comment (yikeke) [inline benchmark/benchmark-tidb-using-tpcc.md:75]:
```suggestion
1. [安装 numactl 工具](https://docs.pingcap.com/zh/tidb/v4.0/check-before-deployment#安装-numactl-工具) 。
```
The link is dead. I replace it with an absolute link. PTAL.

Comment (SE-Bin) [reply benchmark/benchmark-tidb-using-tpcc.md:75]: LGTM

**Pull request merged**